src/main/java/net/pms/dlna/DLNAImageProfile.java

```diff
@@ -1256,22 +1258,10 @@ public DLNAComplianceResult checkCompliance(ImageInfo imageInfo) {
 		}
 
 		switch (this.toInt()) {
-			case DLNAImageProfile.GIF_LRG_INT:
-				checkGIF(imageInfo, complianceResult);
-				break;
-			case DLNAImageProfile.JPEG_LRG_INT:
-			case DLNAImageProfile.JPEG_MED_INT:
-			case DLNAImageProfile.JPEG_RES_H_V_INT:
-			case DLNAImageProfile.JPEG_SM_INT:
-			case DLNAImageProfile.JPEG_TN_INT:
-				checkJPEG(imageInfo, complianceResult);
-				break;
-			case DLNAImageProfile.PNG_LRG_INT:
-			case DLNAImageProfile.PNG_TN_INT:
-				checkPNG(imageInfo, complianceResult);
-				break;
-			default:
-				throw new IllegalStateException("Illegal DLNA media profile");
+			case DLNAImageProfile.GIF_LRG_INT -> checkGIF(imageInfo, complianceResult);
```

Comment:
I find switch statements easier to read the existing way, with one line per case